src/docs/get-started/install/_android-sdk-setup.md

```diff
@@ -0,0 +1,46 @@
+## Android setup (without Android Studio)
```

Comment:
Another thought. Since you weren't able to add "ChromeOS" directly to the install page (at least for now), how about this? Add a note directly to the install page:

```
Looking for Chrome OS instructions?
Check out [Chrome OS install](/docs/get-started/install/chromeos)!
```

Reply:
I was thinking this as well, would you put it in an important box or what type of element would it be wrapped in were you thinking?

Reply:
I'd probably experiment, but maybe start with:

{{site.alert.note}}
  text
{{site.alert.end}}

And see what I thought. I'm a very iterative writer.  :)